tests/qos/test_qos_sai.py

```diff
@@ -1365,6 +1368,12 @@ def testQosSaiPgSharedWatermark(
         if "wm_pg_shared_lossless" in pgProfile:
             pktsNumFillShared = qosConfig[pgProfile]["pkts_num_trig_pfc"]
         elif "wm_pg_shared_lossy" in pgProfile:
+            if not get_src_dst_asic_and_duts['single_asic_test'] and \
```

Comment:
this change has already included in PR https://github.com/sonic-net/sonic-mgmt/pull/8480